packages/gatsby-plugin-sharp/src/duotone.js

```diff
@@ -1,45 +1,44 @@
 const sharp = require(`sharp`)
 
 module.exports = async function duotone(duotone, format, clonedPipeline) {
-  if (duotone) {
-    var duotoneGradient = createDuotoneGradient(
-      hexToRgb(duotone.highlight),
-      hexToRgb(duotone.shadow)
-    )
+  const duotoneGradient = createDuotoneGradient(
+    hexToRgb(duotone.highlight),
+    hexToRgb(duotone.shadow)
+  )
 
-    if (format === `jpg`) {
-      format = `jpeg`
-    }
+  // @todo remove once we upgrade to sharp v0.18 which
+  // adds an alias for "jpg" to toFormat
+  // @see http://sharp.dimens.io/en/stable/changelog/#v0180-30th-may-2017
```

Comment:
This seems like the main blocker for upgrading as we "attention" is the default mode for us for cropping https://github.com/jcupitt/libvips/issues/672#issuecomment-307365416